ReactiveCocoa/Swift/Scheduler.swift

```diff
@@ -9,44 +9,48 @@
 import Foundation
 
 /// Represents a serial queue of work items.
-public protocol SchedulerType {
+public protocol SchedulerProtocol {
 	/// Enqueues an action on the scheduler.
 	///
 	/// When the work is executed depends on the scheduler in use.
 	///
 	/// Optionally returns a disposable that can be used to cancel the work
 	/// before it begins.
-	func schedule(action: () -> Void) -> Disposable?
+	@discardableResult
```

Comment:
I am convinced that in most circumstances, we do not need to cancel the scheduled work (as evidenced by the comment, i.e. "optionally returns"). So if it is not marked with `@discardableResult`, it would mean the call site has to explicitly discard the returned value with `_ =`.